### DISCUSSION TITLE
How does one embed tactics?

### DISCUSSION THREAD

**Discussion by brando90** [princeton-vl/CoqGym#30] [Q&A] [answered]
From discussion https://github.com/princeton-vl/CoqGym/issues/27

**Comment by brando90** [comment, marked as answer]
For great details see: https://github.com/princeton-vl/CoqGym/issues/27 

In short terms are in the gallina language (Coq's functional programming language and more) and tactics are in the tactic language (e.g. ltac, ltac2, mtac...). The original ASTactic model only embeds terms in Gallina as far as I know and produce tactics using the decoder and the grammar specified in the paper.

Feel free to correct me if this is not an accurate summary.

**Comment by yangky11** [comment]
Thanks! I thought there was a button to move an issue to discussions automatically. But I gave up when failing to find the button.

**Reply by brando90** [reply]
makes sense, I've seen a template showing up for gitissues in other projects telling ppl not to post questions as an issue, maybe you can do that? (not sure how but would be nice! :) )

I am happy to help move the other question I wrote, is that ok with you if I go ahead and do it?

**Reply by yangky11** [reply]
Thanks, I'll take a look whenever I get a chance.
Sure, feel free to do it.
